packages/combobox/src/LionCombobox.js

```diff
@@ -850,7 +870,11 @@ export class LionCombobox extends OverlayMixin(LionListbox) {
    */
   _syncToTextboxMultiple(modelValue, oldModelValue = []) {
     const diff = modelValue.filter(x => !oldModelValue.includes(x)).toString();
-    this._setTextboxValue(diff); // or last selected value?
+    const newValue = this.formElements
+      .filter(option => diff.includes(option.choiceValue))
+      .map(option => this._getTextboxValueFromOption(option))
+      .join(' ');
```

Comment:
So multiple choices are concatenated with a space? Can we add a test for that?

Reply:
I've added some tests. But I'm not so sure to understand what is to be done here. I've copy/paste the code from the discussion https://github.com/ing-bank/lion/discussions/1247

Reply:
Before, I think setting an array to the textbox implicitly had the same behavior (concatting with spaces). This makes or  more specific and type compatible. But an extra test never hurts ;)
Maybe you can disable Prettier plugin (on autosave) and add your new tests again?

Reply:
Should be good now for the format. I did some investigation and will open an issue to discuss about it :)